### PR TITLE
Update guestbook chart in repo

### PR DESCRIPTION
**Pull request by hickeyma**
charts/guestbook was updated in a previous PR. This updates the chart to 0.2.0 in the Helm101 repo.

### CODE REVIEW

File: presentation/demosteps.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ $ helm search helm101
 
 Deploy: 
 ```bash
-$ helm install helm101/guestbook --name myguestbook --set serviceType=NodePort
+$ helm install helm101/guestbook --name myguestbook --set service.type=NodePort
```
Comment (spzala) [inline presentation/demosteps.md:29]: Can we keep serviceType=NodePort as a comment that for chart version 0.1.0 we can use it that way?

File: repo/stable/index.yaml [hunk 2]
```diff
@@ -2,11 +2,19 @@ apiVersion: v1
...
+  - apiVersion: v1
+    created: 2019-02-21T14:09:27.964727056Z
+    description: A Helm chart to deploy Guestbook three tier web application.
+    digest: 7f186687f43008ed68e9e4a09aa4e45586e2d6ed5f85287c64bda3f5a7ea4d12
+    name: guestbook
+    urls:
+    - https://ibm.github.io/helm101/guestbook-0.1.0.tgz
     version: 0.1.0
-generated: 2018-05-10T21:50:41.83648523-04:00
+generated: 2019-02-21T14:09:27.909109099Z
```
Comment (spzala) [inline repo/stable/index.yaml:20]: Why we need to change anything in the 0.1.0 section?

File: index.yaml [hunk 3]
```diff
@@ -2,11 +2,19 @@ apiVersion: v1
...
+  - apiVersion: v1
+    created: 2019-02-21T14:09:27.964727056Z
+    description: A Helm chart to deploy Guestbook three tier web application.
+    digest: 7f186687f43008ed68e9e4a09aa4e45586e2d6ed5f85287c64bda3f5a7ea4d12
+    name: guestbook
+    urls:
+    - https://ibm.github.io/helm101/guestbook-0.1.0.tgz
     version: 0.1.0
-generated: 2018-05-10T21:50:41.83648523-04:00
+generated: 2019-02-21T14:09:27.909109099Z
```
Comment (spzala) [inline index.yaml:20]: Why we need to change anything in the 0.1.0 section?

File: presentation/demosteps.md (hunk 1 above)
Comment (hickeyma) [reply presentation/demosteps.md:29]: Sure. I added a note.

File: repo/stable/index.yaml (hunk 2 above)
Comment (hickeyma) [reply repo/stable/index.yaml:20]: This is when `helm index` runs its updates the index file to add the new chart version (0.2.0) and changes the generated timestamp accordingly. I think this probably should changes as it has been regenerated the index file and is not associated to the v0.1.0 timestamp.

I have updated changed the "created" timestamp for chart v0.1.0 back to its original.

File: index.yaml (hunk 3 above)
Comment (hickeyma) [reply index.yaml:20]: Same as above in https://github.com/IBM/helm101/pull/22#discussion_r260414874

**Comment by hickeyma** [comment]
@spzala Thanks for review. Comments updated and ready for review again.

**Comment by spzala** [comment]
Thanks @hickeyma I haven't tested the new chart but the changes lgtm

**Comment by hickeyma** [comment]
Thanks @spzala

**Pull request merged**